.gitignore

```diff
@@ -163,4 +163,7 @@ typings/
 # End of https://www.gitignore.io/api/node,macos,intellij,visualstudiocode
 
 ## documentation
-docs/
\ No newline at end of file
+docs/
+
+## dev build
+dist/prismic-javascript.js
```

Comment:
This will be ignored by npm too, is that what we want to do?

Reply:
I would say yes because we want to have an up to date version any time we use this kit, not some old kept version that does not work with new development